odoo/modules/module.py

```diff
@@ -277,6 +314,9 @@ def module_manifest(path):
             return opj(path, manifest_name)
 
 def read_manifest(addons_path, module):
+    warnings.warn(
+        "read_manifest() is scheduled for removal, please use "
+        "load_manifest() instead.", DeprecationWarning, stacklevel=2)
```

Comment:
just remove it if you want to remove it, it's not part of any API that modules are supposed to use